middlewares/middleware.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  authenticatedOnly: (req, res, next) => {
+    // if session is valid, go to the next stage
+    if (req.session && req.session.currentUser) {
```

Comment:
```suggestion
    if (req?.session?.currentUser) {
```

Reply:
wow, yeah haha, thank you. I  can be shorten this way =))